features/step_definitions/tag_set_steps.rb

```diff
@@ -150,8 +175,18 @@
   visit tag_set_path(tagset)
 end
 
-When(/^I flush the wrangling sidebar caches$/) do
```

Comment:
Is this no longer needed ?

features/tags_and_wrangling/tag_wrangling.feature:    When I flush the wrangling sidebar caches